### ISSUE TITLE
admin file: attributes:preview.php not used?

### ISSUE THREAD

**Issue by torvista** [zencart/zencart#6891]
The only reference to "attributes_preview" is in this link from the category-products, product attribute A infobox:
https://github.com/zencart/zencart/blob/8333810225d0e0ec45c6aa51e97cef1bc0302e5b/admin/category_product_listing.php#L1208

(Although in attributes_controller there is no handler for action "attributes_preview", the attributes are displayed.)

So, what uses admin/includes/attributes_preview.php?

This was prompted (by an unrelated issue in Advanced Shipper) in that this file uses ATTRIBUTES_QTY_PRICE_SYMBOL, which is only defined in the catalog.
So since I can't find a way to call the file, I can't see if this also causes an error.

**Comment by scottcwilson** [comment]
Looks like we stopped using it in 1.5.6.

**Comment by scottcwilson** [comment]
https://github.com/zencart/zencart/commit/a0f066710c21c22c427af425ca426736ca8116f4?w=1

**Comment by drbyte** [comment]
Correct. The attributes_preview.php is not used. I guess it can be deleted.

As far as the UI where the "Attribute Preview" button is displayed:
The Preview button is beside the Edit button.
Clicking Preview takes you to a read-only listing of attributes. (admin-style view, not catalog)
Clicking Edit takes you to the same page, but with top and bottom sections which allow editing selections of attributes as desired.

Is the Preview button problematic/confusing?

UI Options:
1. Leave it as-is.
2. Remove the Preview button.
3. Rename the Preview button to Review.

**Comment by scottcwilson** [comment]
I think this issue just refers to the file, which should be removed.

**Comment by torvista** [comment]
Just to clarify for me and others...as as usual I didn't understand what I'd written.

The file "attributes_preview" is not accessible by anything. Period. If it does not offer anything, then it should be removed.

The category-product listing offers the A option on a product which then shows an infobox with a "Preview" option for a product with attributes. That Preview is generated in attributes_controller.

I don't think that's confusing. Whether there is any point to it is another issue...it has no Cancel or Back or Edit, just a dead end.
Strictly speaking it is a review, not a preview.
